server/src/main/java/org/apache/druid/server/security/ResourceType.java

```diff
@@ -34,6 +34,7 @@
   public static final String STATE = "STATE";
   public static final String SYSTEM_TABLE = "SYSTEM_TABLE";
   public static final String QUERY_CONTEXT = "QUERY_CONTEXT";
+  public static final String EXTERNAL = "EXTERNAL";
```

Comment:
We might want to add this perm to the security-user-auth.md

Reply:
Right now, no production code actually uses this permission, so I had left it out of docs. Definitely we should document it when it is used in production code.